commcare_connect/templates/opportunity/user_visit_details.html

```
@@ -301,43 +298,41 @@ <h2 class="title">{% translate "Reason for Rejection" %}</h2>
+           x-transition.opacity
+           class="modal-backdrop">
+        <div @click.away="showApproveModal = false" x-transition class="modal">
+          <div class="header flex justify-between items-center">
+            <h2 class="title">{% translate "Justification for Approval" %}</h2>
+            <button @click="showApproveModal = false"
+                    class="button-icon"
+                    aria-label="Close">
+              <i class="fa-solid fa-xmark"></i>
+            </button>
+          </div>
+          <form hx-post="{% url "opportunity:approve_visits" org_slug=request.org.slug opp_id=user_visit.opportunity.opportunity_id %}"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)